### PR TITLE
finds stores based on actual route time

### PR DESCRIPTION
**Pull request by hannahli23**
Changes
----

- Changed app.js back for easy to test purposes
- Changed map.js to return stores to the frontend based on actual route's time

Sample API response:
```json
{
    "ok": true,
    "results": [
        {
            "name": "Walmart Supercenter",
            "location": {
                "lat": 33.58701110000001,
                "lng": -84.4720621
            },
            "id": "ChIJ3765IH3j9IgRYCN1rrWI140",
            "vicinity": "6149 Old National Highway, College Park",
            "distance": {
                "text": "17.0 mi",
                "value": 27354
            },
            "duration": {
                "text": "26 mins",
                "value": 1584
            }
        },
        {
            "name": "ALDI",
            "location": {
                "lat": 33.4724338,
                "lng": -84.44463619999999
            },
            "id": "ChIJWRZMROfv9IgRIua9mO_7RFA",
            "vicinity": "1060 Georgia 85 N, Fayetteville",
            "distance": {
                "text": "9.9 mi",
                "value": 15942
            },
            "duration": {
                "text": "19 mins",
                "value": 1144
            }
        },
        {
            "name": "Walmart Supercenter",
            "location": {
                "lat": 33.5732,
                "lng": -84.5416632
            },
            "id": "ChIJO56STL_m9IgRftk7bIqt2Lc",
            "vicinity": "4735 Jonesboro Road, Union City",
            "distance": {
                "text": "13.3 mi",
                "value": 21376
            },
            "duration": {
                "text": "23 mins",
                "value": 1361
            }
        },
        {
            "name": "ALDI",
            "location": {
                "lat": 33.4418943,
                "lng": -84.322878
            },
            "id": "ChIJX7zpyuL29IgRptD3eaYlZ14",
            "vicinity": "11499 Tara Boulevard, Lovejoy",
            "distance": {
                "text": "16.3 mi",
                "value": 26294
            },
            "duration": {
                "text": "30 mins",
                "value": 1782
            }
        },
        {
            "name": "The Fresh Market",
            "location": {
                "lat": 33.409364,
                "lng": -84.569181
            },
            "id": "ChIJR0B2Hirq9IgRjXqqqSC6vi0",
            "vicinity": "100 North Peachtree Parkway, Peachtree City",
            "distance": {
                "text": "1.2 mi",
                "value": 1949
            },
            "duration": {
                "text": "4 mins",
                "value": 237
            }
        }
    ]
}
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,5 +22,6 @@
     "danger": "^10.1.0"
   },
   "dependencies": {
+    "google-distance-matrix": "^1.1.1"
```
Comment (evan10s) [inline package.json]: This should be a server dependency

File: server/app.js [hunk 2]
```diff
@@ -11,29 +11,24 @@ const app = express();
 
 // view engine setup
 app.set("views", path.join(__dirname, "views"));
-app.set("view engine", "ejs");
+app.set("view engine", "jade");
```
Comment (evan10s) [inline server/app.js]: This should be ejs

File: server/app.js [hunk 3]
```diff
@@ -11,29 +11,24 @@ const app = express();
...
 app.use(cors());
 app.use(logger("dev"));
 app.use(express.json());
-app.use(express.urlencoded({extended: false}));
+app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
-app.use(express.static(path.join(__dirname, "../client/build")));
 app.use(express.static(path.join(__dirname, "public")));
 
-app.use("/api/v1/map", mapRouter);
```
Comment (evan10s) [inline server/app.js]: All the routing changes in this file should be reverted so the frontend is served properly

**Review by evan10s — CHANGES_REQUESTED** [review]
I haven't tested it, but undoing most of the changes in server/app.js should get it building again in CI

**Comment by 9353oasis** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Files in the [`client`](/client) directory were modified; please update the client version.
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Files in the [`server`](/server) directory were modified; please update the server version.
  </td>
    </tr>
  </tbody>
</table>



<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>ℹ</td>
      <td>This pull request includes client dependency changes.</td>
    </tr>
  
<tr>
      <td>ℹ</td>
      <td>This pull request includes server dependency changes.</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against a4636a4f1e08e7b9b21e5cee88fdec35ef3bafb2
</p>

**Comment by 9353oasis** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>ℹ</td>
      <td>This pull request includes server dependency changes.</td>
    </tr>
  
<tr>
      <td>ℹ</td>
      <td>This pull request includes client dependency changes.</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 58d376a63155d3edd7db91da1266b13b278e32d7
</p>

**Review by evan10s — APPROVED** [review]

**Comment by 9353oasis** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>ℹ</td>
      <td>This pull request includes server dependency changes.</td>
    </tr>
  
<tr>
      <td>ℹ</td>
      <td>This pull request includes client dependency changes.</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against f8506828383ca393f7add46fadac0b94db06cfb3
</p>

**Pull request merged**